packages/pangea-sdk/pangea/services/base.py

```diff
@@ -1,32 +1,65 @@
 # Copyright 2022 Pangea Cyber Corporation
 # Author: Pangea Cyber Corporation
 
+import logging
+from dataclasses import dataclass
+from logging.handlers import TimedRotatingFileHandler
 from typing import Optional
 
 from pangea import __version__
 from pangea.config import PangeaConfig
 from pangea.request import PangeaRequest
 
 
+@dataclass
+class LoggerConfig:
+    # If set logger, SDK will use this logger without any extra configuration to log
+    logger: Optional[logging.Logger] = None
+
+    # If set to True, SDK will user default logger configuration (Ignored if logger is set up)
+    logger_enable_default: bool = False
+
+    # Set to choose logger name (if None, service_name is used by default)
+    logger_name: Optional[str] = None
+
+
 class ServiceBase(object):
     service_name: str = "base"
     version: str = "v1"
 
-    def __init__(self, token, config: Optional[PangeaConfig] = None):
+    def __init__(self, token, config: Optional[PangeaConfig] = None, logger_config: Optional[LoggerConfig] = None):
```

Comment:
where/when is logger_config passed in?

Reply:
In the constructor, when user create service class to make request to the API